src/store/modules/channel/mutations.ts

```diff
@@ -0,0 +1,13 @@
+import { MutationTree } from 'vuex'
+import ChannelState from './ChannelState';
+import * as types from './mutation-types'
+
+const mutations: MutationTree <ChannelState> = {
+  [types.CHANNEL_INVENTORY_CHANNELS_UPDATED] (state, payload) {
+    state.inventoryChannels = payload;
+  },
+  [types.CHANNEL_CLEARED](state) {
```

Comment:
No need to have a separate mutation to clear the state we can use `CHANNEL_INVENTORY_CHANNELS_UPDATED` mutation.